plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityCharging.java

```diff
@@ -0,0 +1,79 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.entity.Vex;
+
+public class EntityCharging implements Property {
+
+    public static boolean describes(ObjectTag entity) {
+        return entity instanceof EntityTag
+                && ((EntityTag) entity).getBukkitEntity() instanceof Vex;
+    }
+
+    public static EntityCharging getFrom(ObjectTag entity) {
+        if (!describes(entity)) {
+            return null;
+        }
+        else {
+            return new EntityCharging((EntityTag) entity);
+        }
+    }
+
+    public static final String[] handledMechs = new String[] {
+        "charging"
+    };
+
+    private EntityCharging(EntityTag _entity) {
+        entity = _entity;
+    }
+
+    EntityTag entity;
+
+    public static void registerTags() {
+
+        // <--[tag]
+        // @attribute <EntityTag.charging>
```

Comment:
This should be merged into EntityTag.charged.

Reply:
merged